ADOPTERS.md

```diff
@@ -0,0 +1,21 @@
+# Kubeapps Adopters
+
+If you're using Kubeapps and want to add your organization to this list, follow these directions!
+
+## Organizations using Kubeapps
+
+![logo](./docs/logos/vmware-tanzu.svg)
```

Comment:
If you check the result at https://github.com/ppbaena/kubeapps/blob/oss-documentation-update/ADOPTERS.md , you'll see that as an SVG with no dimensions, it's just going to scale to width, which we may not want. AFAICT, that means using an img element instead. See https://gist.github.com/absoludity/739192bb1d6da37faf1329f81d1fb568 for an example.


Also, I think the text `logo` should clearly identify the organization instead?